pkg/operators/machine-api-operator.go

```diff
@@ -24,289 +24,298 @@ var (
 	maoManagedDeployment = "machine-api-controllers"
 )
 
-var _ = Describe("[Feature:Operators][Disruptive] Machine API operator deployment should", func() {
-	defer GinkgoRecover()
+var _ = Describe(
+	"[Serial][Feature:Operators][Disruptive] Machine API operator deployment should",
```

Comment:
Should the prefixes be dropped from these?
```suggestion
	"Machine API operator deployment should",
```